markup/pandoc/convert.go

```diff
@@ -77,6 +80,64 @@ func getPandocBinaryName() string {
 	return ""
 }
 
+const citeprocBinary = "pandoc-citeproc"
+
+func getCiteprocBinaryName() string {
+	if hexec.InPath(citeprocBinary) {
+		return citeprocBinary
+	}
+	return ""
+}
+
+func getCiteprocArgs(cfg converter.ProviderConfig) []string {
+	pandocVersion, err := getPandocVersion(cfg)
```

Comment:
I think it's safe to assume that the pandoc version can be cached (e.g. via a `sync.Once`). Shelling out to `os/exec` isn't cheap.

Reply:
Done!